frontend/src/bundles/common/components/chip/chip.tsx

```diff
@@ -0,0 +1,28 @@
+import { Close } from '@mui/icons-material';
+import { Chip as MUIChip } from '@mui/material';
+
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+    label: string;
+    onDelete?: () => void;
+};
+
+const Chip = ({ label, onDelete }: Properties): JSX.Element => {
```

Comment:
We use React.FC in our components, maybe better to make it in one style?

Reply:
It's a good question, than we need to change chip, input, loader, textarea. Should we do it in bug fix ?

Reply:
I don't think this should be a bug fix since it is not a bug, and PR is not merged yet. That is my opinion though, not sure how others feel.

Reply:
okay, I'll change it

Reply:
> It's a good question, than we need to change chip, input, loader, textarea. Should we do it in bug fix ?

I got it, yes, we need to make `React.FC`. Could you create a bug ticket?